applications/MedApplication/python_registry_lists.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+
+python_modelers_to_be_registered: List[str] = ["modelers.import_med_modeler.ImportMedModeler"]
```

Comment:
```suggestion
python_modelers_to_be_registered: List[str] = ["import_med_modeler.ImportMedModeler"]
```
As it is in the modelers list it will automatically add the corresponding `modeler` keyword. Long story short, in the list it is only required to add `python_module.ClassName`.

Reply:
I copied it from the core where it is done like I do
https://github.com/KratosMultiphysics/Kratos/blob/5bf5e40b7fc6e061e5e2f3056efda3ed3b59d963/kratos/python_interface/python_registry_lists.py#L3

I prefer to keep it consistent

Reply:
The point is that in the core we have a submodule `modeler` coming from the corresponding subfolder within `python_scripts`. I didn't note that this is also the case in the `MedApplication` so you're right.  Forget about my comment (I'd leave it as unresolved as "documentation").